app/lib/Forms.js

```diff
@@ -35,9 +35,9 @@ async function save(input,collection,req)
   //
   // req must contain user and ip fields.
   //
-  // collection is a string, ususally "testForms" that indicates which collection this is.
+  // collection is a string that indicates which collection this is.
 
-  collection = collection || "testForms";
+  collection = collection;
```

Comment:
Removed.  Will roll this change into the next PR, if that's okay - since I'm already making other changes to this file, and it doesn't seem to be doing any harm being here for the moment.